src/main/java/com/exadel/discount/controller/DiscountController.java

```diff
@@ -52,4 +51,16 @@ public DiscountDTO addDiscount(@RequestBody @Valid CreateDiscountDTO discountDTO
     public void deleteDiscount(@PathVariable(name = "id") @NotNull UUID id) {
         discountService.deleteById(id);
```

Comment:
Should here be a check if discount with such id exist?

Reply:
Yes, it is in service